net/grpc/gateway/examples/echo/envoy.yaml

```diff
@@ -35,9 +35,15 @@ static_resources:
                 max_age: "1728000"
                 expose_headers: custom-header-1,grpc-status,grpc-message
           http_filters:
-          - name: envoy.filters.http.grpc_web
-          - name: envoy.filters.http.cors
-          - name: envoy.filters.http.router
+            - name: envoy.filters.http.grpc_web
```

Comment:
just curious - does the name still matter now?

Reply:
I've checked. Looks like the name doesn't matter anymore.

Reply:
Oh i see! Good to know.. Thanks for checking! :)